styles.css

```diff
@@ -0,0 +1,960 @@
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.role-year {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 0;
+  top: 8px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 0.5px;
+}
+
+.project-description {
+  margin-left: 5%;
+}
+
+.project-description > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.project-details {
+  width: 70%;
+  height: 76px;
+  left: 0;
+  top: 92px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills {
+  display: flex;
+  float: right;
+  align-items: center;
+  position: static;
+  padding: 0.5px;
+  width: 100%;
+  height: 24px;
+  left: 0.5px;
+  top: 180px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: 10%;
+}
+
+li.skill {
+  display: inline-block;
+  padding: 4px 12px;
+  position: static;
+  width: 53px;
+  height: 24px;
+  left: 0.5px;
+  top: 0.5px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.5px 8px;
+}
+
+.skill {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.button-click {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin-top: 30%;
+}
+
+.button-resume {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+  padding-left: 20px;
+}
+
+.btn {
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  padding: 7px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+.about {
+  background: #fff;
+  width: 90%;
+  margin-left: 25px;
+  margin-top: 30px;
+  border-top-right-radius: 50px;
+}
+
+.about-me {
+  display: flex;
+  position: static;
+  width: 98%;
+  padding: 0.5px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40.5px;
+  line-height: 52px;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-top: 60px;
+}
```

Comment:
- [ ] Nice work adding the round corner at the top right side as requested by the previous reviewer :100: But the space at the top of the title "About Myself" it missing. You can add a `padding-top` to the `.about-me` class to fix that.

### Your design
![228537306-d0585e76-0be9-42aa-be98-9c7b42901a0d](https://user-images.githubusercontent.com/91142432/228585190-b02e2a92-0d01-40e1-8801-d7e5ca8fb965.png)


### Figma's design
![228537150-fee121ac-cbc6-49ca-83cf-7930441843d3](https://user-images.githubusercontent.com/91142432/228585003-7f8a1f92-751a-4362-a254-26c79762e210.png)